lib/dynosaur/inputs/new_relic_input_plugin.rb

```diff
@@ -0,0 +1,63 @@
+require 'dynosaur/new_relic_api_client'
+require 'dynosaur/version'
+require 'dynosaur/error_handler'
+
+# ScalerPlugin implementation that uses the New Relic API
+# to get the current requests per minute on the site, and scale to an
+# appropriate number of dynos.
+
+module Dynosaur
+  module Inputs
+    class NewRelicInputPlugin < AbstractInputPlugin
+
+      # Load config from the config hash
+      def initialize(config)
+        super
+
+        @unit = "RPM"
+        @key = config["key"]
+        @app_id = config["app_id"].to_s
+        @rpm_per_dyno = config["rpm_per_dyno"].to_i
+
+        if @key.blank?
+          raise "You must supply API key in the new relic plugin config"
+        end
+        if @app_id.blank?
+          raise "You must supply appid in the new relic plugin config"
+        end
+
+        @metric_name = "HttpDispatcher"
+        @new_relic_api_client = Dynosaur::NewRelicApiClient.new(@key, @app_id)
+      end
+
+      def retrieve
+        return get_rpm
+      end
+
+      def value_to_resources(value)
+        estimate = (value / @rpm_per_dyno.to_f).ceil
+        puts "NEW RELIC: #{@recent} => #{value} : #{estimate}"
```

Comment:
@werkshy Should we log this? This was in the previous version of the plugin, but it doesn't seem like we do it in the GA plugin.